### PR TITLE
Add support for WriteableBitmap

### PR DESCRIPTION
**Pull request by jeromelaban**

### CODE REVIEW

File: doc/articles/supported-features.md [hunk 1]
```diff
@@ -135,6 +135,7 @@
 - XAML Reader
 - Phased Binding (x:Phase)
 - XAML code-behind Events registration
+- WriteableBitmap
```
Comment (carldebilly) [inline doc/articles/supported-features.md]: The list should have an order...

File: src/Uno.UI.Wasm/WasmScripts/Uno.UI.d.ts [hunk 2]
```diff
@@ -251,6 +251,19 @@ declare module Uno.UI.Interop {
         main_class: Interop.IMonoClassHandle;
     }
 }
+declare module Uno.UI.Interop {
+    interface IWebAssemblyModule {
```
Comment (carldebilly) [inline src/Uno.UI.Wasm/WasmScripts/Uno.UI.d.ts]: The term `Module` is already used in TS/JS for something else :-|

Comment (jeromelaban) [reply src/Uno.UI.Wasm/WasmScripts/Uno.UI.d.ts]: True, but emscripten uses this already... I'm not sure I can change it.

File: src/Uno.UI.Wasm/ts/Interop/_global.ts [hunk 3]
```diff
@@ -3,4 +3,5 @@
 // Those variables are created in ooui.js, from the framework
```
Comment (Dissolute) [inline src/Uno.UI.Wasm/ts/Interop/_global.ts]: "Those variables are created in ooui.js" - copy-paste?

File: src/Uno.UI/UI/Xaml/Controls/Image/Image.Android.cs [hunk 4]
```diff
@@ -444,6 +450,29 @@ private void SetSourceBitmap(ImageSource newImageSource)
 				var unused = SetSourceBitmapAsync(CancellationToken.None, newImageSource);
 			}
 		}
+
+		private void SetFromWriteableBitmap(WriteableBitmap bitmap)
+		{
+			var drawableBuffer = new int[bitmap.PixelWidth * bitmap.PixelHeight];
+			var sourceBuffer = (bitmap.PixelBuffer as InMemoryBuffer).Data;
+			for (int i = 0; i < drawableBuffer.Length; i++)
```
Comment (Dissolute) [inline src/Uno.UI/UI/Xaml/Controls/Image/Image.Android.cs]: Bit of green wanted here

**Pull request merged**